src/create.wsgi

```diff
@@ -199,7 +199,8 @@ def application(environ, start_response):
                             _("Required file '%s' is missing") % required_file)
 
     if task.get_type() in [TASK_VMCORE, TASK_VMCORE_INTERACTIVE]:
-        task.strip_vmcore(os.path.join(crashdir, "vmcore"))
+        vmcore = KernelVMcore(os.path.join(crashdir, "vmcore"))
+        vmcore.strip_extra_pages(task)
```

Comment:
I suppose this now needs `vmcore.prepare_debuginfo(task)` beforehand?

Reply:
And to drop the argument from the `strip_extra_pages()` call.

Reply:
Indeed. Missed that in the review.

Reply:
Nice catch!  I need a test for this. Do we have a tool that calls this method?  I used “retrace-server-task create” and the manager page.  But neither of these calls this create code.

Reply:
Good question, I thought `retrace-server-task create` would hit the path? It actually did so in my testing.

Reply:
I can look at the traces later to see if it reveals something more.

Reply:
Ok I see it now.  There is 3 options for 'retrace-server-task create':
1. 'manager' - uses '__custom__' which is default and well tested, and references a file local to retrace-server.
2. 'ftp' - uses 'start' method.  This is non-default but is lightly used because it fetches the vmcore from ftp.
3. 'http' - this uses 'create' method.  This is non-default, and you have to send the vmcore over http.  It is not used much if at all by us so this explains why I missed it.  I'll need to review it and test.

A quick review and it looks like this definitely needs a closer look so I'll see what else may be needed.  Even with 7211122c4cf3e704cbab68a6767c7389150c0939 we are not handling flattened format for example via the 'create' code path.  I am not sure if I can merge the logic inside 'start_vmcore' with what is inside 'create' or if it truly needs duplicating.